controllers/userController.js

```diff
@@ -9,11 +9,13 @@
  */
 
```

Comment:
To get the model now its just
`const connectDB = require('../config/dbConn');`
`const sequelize = connectDB();`
`const { Users } = sequelize.models;`